src/hexo/themes/documentation/source/images/results-interop.svg

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 71.8 72" style="enable-background:new 0 0 71.8 72;" xml:space="preserve">
+<style type="text/css">
+	.st0{opacity:0.85;fill:#5753B4;enable-background:new    ;}
+	.st1{opacity:0.85;fill:#817EC7;enable-background:new    ;}
+	.st2{opacity:0.53;fill:#0B0687;enable-background:new    ;}
+	.st3{fill:#ACADDA;}
+</style>
+<title>icons</title>
+<g>
+	<g id="Layer_1_1_">
+		<circle class="st0" cx="17.2" cy="43.7" r="14"/>
+		<circle class="st1" cx="27.5" cy="28.2" r="13.9"/>
+		<circle class="st2" cx="36.9" cy="44" r="14"/>
+		<path class="st3" d="M44.5,28.3c0-9.5-7.7-17.2-17.2-17.3s-17.2,7.7-17.3,17.2l0,0C1.5,32.1-2.4,42.3,1.5,50.9s14.1,12.5,22.8,8.6
+			c1-0.5,2-1,2.9-1.7c7.8,5.5,18.5,3.6,24-4.2s3.6-18.5-4.2-24C46.2,29.2,45.4,28.7,44.5,28.3z M34.2,40.3c-0.7-3.1-2.2-6-4.4-8.3
+			c3.3-2.1,7.4-2.6,11.2-1.5C40.3,34.6,37.8,38.2,34.2,40.3z M27.2,53.5c-2.3-2.3-3.6-5.3-3.9-8.5c2.6,0.6,5.3,0.6,7.8,0.1
+			C30.8,48.2,29.4,51.2,27.2,53.5z M27.3,42.1c-1.3,0-2.7-0.2-3.9-0.6c0.5-2.8,1.8-5.4,3.8-7.5c2,2.1,3.4,4.7,3.8,7.5
+			C29.8,41.9,28.6,42.1,27.3,42.1z M20.2,40.1c-3.5-2.1-6-5.7-6.6-9.7c3.7-1,7.7-0.4,10.9,1.6C22.4,34.3,20.9,37.1,20.2,40.1z
+			 M27.3,14.4c7.2,0,13.2,5.5,13.8,12.6c-4.8-1.2-9.9-0.2-14,2.7c-4-2.8-8.9-3.8-13.7-2.7C14.2,19.9,20.1,14.4,27.3,14.4z M3.4,43.8
+			c0-5,2.7-9.6,7-12c1.1,5.2,4.6,9.7,9.4,12c0,4.4,1.7,8.6,4.7,11.8c-6.5,4-15.1,1.9-19.1-4.6C4.1,48.8,3.4,46.3,3.4,43.8z
+			 M37.1,57.6c-2.6,0-5.1-0.7-7.3-2.1c3-3.2,4.7-7.4,4.7-11.7c4.9-2.3,8.5-6.7,9.6-12c6.6,3.9,8.8,12.4,4.8,19
+			C46.5,55.1,41.9,57.6,37.1,57.6L37.1,57.6z"/>
+	</g>
+</g>
+</svg>
```

Comment:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 71.8 72"><style>.a{opacity:.85;fill:#5753b4}.b{opacity:.85;fill:#817ec7}.c{opacity:.53;fill:#0b0687}.d{fill:#acadda}</style><circle cx="17.2" cy="43.7" r="14" class="a"/><circle cx="27.5" cy="28.2" r="13.9" class="b"/><circle cx="36.9" cy="44" r="14" class="c"/><path d="M44.5 28.3c0-9.5-7.7-17.2-17.2-17.3S10.1 18.7 10 28.2C1.5 32.1-2.4 42.3 1.5 50.9s14.1 12.5 22.8 8.6c1-.5 2-1 2.9-1.7 7.8 5.5 18.5 3.6 24-4.2s3.6-18.5-4.2-24c-.8-.4-1.6-.9-2.5-1.3zm-10.3 12c-.7-3.1-2.2-6-4.4-8.3 3.3-2.1 7.4-2.6 11.2-1.5-.7 4.1-3.2 7.7-6.8 9.8zm-7 13.2c-2.3-2.3-3.6-5.3-3.9-8.5 2.6.6 5.3.6 7.8.1-.3 3.1-1.7 6.1-3.9 8.4zm.1-11.4c-1.3 0-2.7-.2-3.9-.6.5-2.8 1.8-5.4 3.8-7.5 2 2.1 3.4 4.7 3.8 7.5-1.2.4-2.4.6-3.7.6zm-7.1-2c-3.5-2.1-6-5.7-6.6-9.7 3.7-1 7.7-.4 10.9 1.6-2.1 2.3-3.6 5.1-4.3 8.1zm7.1-25.7c7.2 0 13.2 5.5 13.8 12.6-4.8-1.2-9.9-.2-14 2.7-4-2.8-8.9-3.8-13.7-2.7.8-7.1 6.7-12.6 13.9-12.6zM3.4 43.8c0-5 2.7-9.6 7-12 1.1 5.2 4.6 9.7 9.4 12 0 4.4 1.7 8.6 4.7 11.8-6.5 4-15.1 1.9-19.1-4.6-1.3-2.2-2-4.7-2-7.2zm33.7 13.8c-2.6 0-5.1-.7-7.3-2.1 3-3.2 4.7-7.4 4.7-11.7 4.9-2.3 8.5-6.7 9.6-12 6.6 3.9 8.8 12.4 4.8 19-2.4 4.3-7 6.8-11.8 6.8z" class="d"/></svg>
```

Reply:
@qzhou1607 can you update this file with the content of @alrra's comment?